packages/protocol/contracts/layer1/based/TaikoInbox.sol

```diff
@@ -372,6 +373,10 @@ abstract contract TaikoInbox is EssentialContract, ITaikoInbox, ITaiko {
         return true;
     }
 
+    function isForkActive() external view override returns (bool) {
+        return state.stats2.numBlocks >= getConfigV3().forkHeights.pacaya;
```

Comment:
Looks like this value will be changed to `numBatches`? Then we can't know the exact block numbers in protocol anymore?

Reply:
This is based on the previous PR, once the batch PR is merged, this var will be renamed.